lib/src/util/__tests__/useFuzzyFilter.test.tsx

```diff
@@ -51,50 +45,129 @@ describe('useFuzzyFilter', function () {
     threshold: 0.1
   };
 
-  const getShallow = getShallowComponent<Props>(HookWrapper, {input, options});
+  // since hooks can only be used inside a function component we wrap it inside one
+  function FixtureWithFilter({ input, options }: Props) {
+    const [output, filterTerm, setFilterTerm] = useFuzzyFilter(input, options);
+
+    function onChange(e: ChangeEvent<HTMLInputElement>) {
+      setFilterTerm(e.target.value);
+    }
+
+    return (
+      <>
+        <input onChange={onChange} value={filterTerm.toString()}/>
+        { output.map((result) => <span data-testid="result" key={result.id}>{result.name}</span>) }
+      </>
+    );
+  }
+
+  const renderWithFilter = rtlRender<Props>(FixtureWithFilter, {input, options});
 
   it('returns unchanged input if filter was not set', function () {
-    const shallowRender = getShallow();
-    const [output, filter] = shallowRender.prop('data-result');
-    expect(output).toBe(input);
-    expect(filter).toBe('');
+    function FixtureNoFilter() {
+      const [output] = useFuzzyFilter(input, options);
+      return (
+        <>
+          { output.map((result) => <span key={result.id} data-testid="result">{result.name} </span>) }
+        </>
+      );
+    }
+
+    const view = render(<FixtureNoFilter />),
+        outputItems = view.getAllByTestId('result');
+
+    expect(outputItems.length).toBe(4);
+    expect(outputItems[0]).toHaveTextContent('Bicycle');
+    expect(outputItems[1]).toHaveTextContent('Motorcycle');
+    expect(outputItems[2]).toHaveTextContent('Skateboard');
+    expect(outputItems[3]).toHaveTextContent('Moped');
   });
 
   it('returns unchanged input if filter was set to empty string', function () {
```

Comment:
I'm unsure if this test is working as intended. useFuzzyFilter initially sets filterTerm as an empty string, so I don't THINK setFilterTerm needs to be called for this test? But if that's the case, is this simply the same test as the before it?

Reply:
> I'm unsure if this test is working as intended. useFuzzyFilter initially sets filterTerm as an empty string, so I don't THINK setFilterTerm needs to be called for this test? But if that's the case, is this simply the same test as the before it?

I'm in favor of removing this test. Looks like either way we're testing the same "filter is not set" vs "filter is empty", which basically looks to me, like the same thing.